website/content/docs/templates/hcl_templates/blocks/packer.mdx

```diff
@@ -66,7 +66,73 @@ packer {
 }
 ```
 
-For more information, see [Plugins](/packer/docs/plugins).
+Using the `required_plugins` block to codify the plugins required for invoking a 
+`packer build` on your template(s) is a great way to ensure consistent builds across
+different platforms or hosts.
+
+The block codifies two pieces of information that we've found critical to ensure reproducible builds:
+1. **The source of a required plugin** indicates to users where a plugin where a plugin was downloaded from
+and where to reach out if there are issues with the plugin. 
+1. **The version of a required plugin** indicates to users the exact or minimum version needed for the build. 
+This is a great way to pin approved versions of a plugin that can be installed across your environment or team.
+
+For more information on plugins, refer to [Plugins](/packer/docs/plugins).
+
+## Source Addresses
+
+A plugin's source address is its global identifier that provides users codified information on where a plugin is sourced and can be obtained. 
+When used in conjunction with the `packer init` command a source address tells Packer where to download the remote plugin from. 
+
+At minimum source addresses consist of three parts delimited by slashes (`/`), as
```

Comment:
Oh is 2 something we've supported previously?

Reply:
Nope, this is part of the `relaxed` bundle, beforehand it was 3 and only 3, so I picked 2 as the new minimum while at it, I also considered just a host even, but maybe that's a stretch?

Reply:
Can you please help me understand the rationale for supporting 2? 

For Packer 1.7, we originally supported only 2, but landed on it being at minimum hostname/namespace/repo to mitigate against potential username or organizational name squatting on GitHub.

At a minimum, the "namespace/plugin" is needed as that is the convention used by Terraform and Vagrant. Since Packer relies on a publicly accessible registry (GitHub) for hosting plugins, we cannot assume a default hostname like Terraform or Vagrant to mitigate against potential username or organizational name squatting. 

As we continue improving remote plugin installation, we may find use cases for relaxing the source address further. I don't think we have enough information at this time. Please let me know if I am mistaken.

Reply:
Honestly it's a pretty arbitrary lower-bound, no particular rationale to support it besides why not, hence why I also mention hesitating to have one level as well at this point.
If there are genuine security concerns regarding the minimum being 2 parts to that source URI (given that Github only is supported and we explicitely require a 3-part URI for this case, I'd be surprised if it became an issue), we can go back to a minimum of 3, I don't have any objection to that. Besides, in practice even when we do support more remote sources, I'd be surprised if any of the realistic sources required paths with as few parts to their base URI so it would be reasonable to leave that requirement as-is.

Reply:
Let's go back to a minimum of 3 in a separate PR. 

If there are no other objections to these docs I will merge as is. Let me know.